src/expandableCalendar/agendaList.js

```diff
@@ -109,7 +109,7 @@ class AgendaList extends Component {
 
     if (markToday) {
       const todayString = XDate.locales[XDate.defaultLocale].today || commons.todayString;
-      const isToday = dateutils.isToday(XDate(title));
+      const isToday = dateutils_isToday(XDate(title));
```

Comment:
Again, better change the name of the const instead of changing the name of the function